File: PHP/uberVan.php
<?php

require_once('car.php');

class UberVan extends Car {
  private $typeAccepted;
  private $seatMaterial;

  public function __construct($license, $driver, $typeAccepted, $seatMaterial) {
    parent::__construct($license, $driver);
    $this->typeAccepted = $typeAccepted;
    $this->seatMaterial = $seatMaterial;
  }
}

File: java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car {
  Map<String, ArrayList<Integer>> typeAccepted;
  ArrayList<String> SeatMaterial;

  public UberVan(String License, Account driver, Map<String, ArrayList<Integer>> typeAccepted, ArrayList<String> SeatMaterial) {
    super(License, driver);
    this.typeAccepted = typeAccepted;
    this.SeatMaterial = SeatMaterial;
  }
}
